### PR TITLE
SwiftGraphQL blog post

### PR DESCRIPTION
**Pull request by maticzav**
Hi 👋 ,

This is a draft for the blog post about SwiftGraphQL. It's still missing the examples code because I started working on refining the API once I reached that point, but it feels like it might be better to ship as it is right now.

How does it feel when you are reading - is the tone inviting? Do you think I should add more focus on one of the topics?

### CODE REVIEW

File: pages/blog/swift-graphql.mdx [hunk 1]
```diff
@@ -0,0 +1,160 @@
+import withArticle from '../../ui/blog/article';
+
+<!-- Configuration -->
+
+export default withArticle({
+  title: 'SwiftGraphQL - A GraphQL client that lets you forget about GraphQL',
+  author: 'maticzav',
+  tags: ['graphql', 'swift', 'client', 'swift-graphql'],
+  date: '2021-03-25',
+  description: 'An introduction article for SwiftGraphQL client.',
```
Comment (Urigo) [inline pages/blog/swift-graphql.mdx:10]: maybe something sexier?
LIke "A new GraphQL Client for Swift Lovers" or something....

File: pages/blog/swift-graphql.mdx [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+  image:
+    'https://github.com/the-guild-org/oneblog/blob/master/img/704/1*emrpS_BYaLltTGadD6GaIw.png?raw=true',
```
Comment (Urigo) [inline pages/blog/swift-graphql.mdx:12]: Could you ask Idan on our Slack for get you a nice image?

File: pages/blog/swift-graphql.mdx [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+        print(data.human)
+    }
+}
+```
+
+Easy, right?
+
+## Beyond the basics
+
+The example above only scratched the surface of what is possible with SwiftGraphQL. To give you an idea of what you could do, consider these two examples.
```
Comment (Urigo) [inline pages/blog/swift-graphql.mdx:103]: every time you mention SwiftGraphQL, put it with a link to the website/repo

File: pages/blog/swift-graphql.mdx [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+- You can write queries programmatically,
+- It supports queries, mutations as well as subscriptions.
+
+And the best part? It's super easy to get started.
+
+1. Install the generator using `brew install swift-graphql`,
+1. Generate the API by running `swift-graphql <endpoint> --output ./file.swift`,
+1. Start querying your data.
+
+SwiftGraphQL is a young, slowly evolving library. I want to make it the best Swift GraphQL client; that's why I'd love to hear how you use it. To support its development, make sure you leave a star on GitHub and connect with me on Twitter. And if you are actively using it, consider becoming my sponsor on GitHub.
```
Comment (Urigo) [inline pages/blog/swift-graphql.mdx:158]: Add links every time you mention somewhere to go

**Review by Urigo — CHANGES_REQUESTED** [review]
@maticzav great post!
I've added very small ideas, you don't have to use them...

other then that we can publish it whenever you're ready!

File: pages/blog/swift-graphql.mdx (hunk 2 above)
Comment (maticzav) [reply pages/blog/swift-graphql.mdx]: I like the one Idan made, I think you were thinking of something similar, right?

File: pages/blog/swift-graphql.mdx [hunk 5]
```diff
@@ -0,0 +1,118 @@
+import withArticle from '../../ui/blog/article';
+
+<!-- Configuration -->
+
+export default withArticle({
+  title: 'SwiftGraphQL - A GraphQL client for Swift lovers.',
+  author: 'maticzav',
+  tags: ['graphql', 'swift', 'client', 'swift-graphql'],
+  date: '2021-07-24',
+  description: 'SwiftGraphQL is a GraphQL client that focuses on ',
```
Comment (Urigo) [inline pages/blog/swift-graphql.mdx:10]: missing the end of the sentence

**Review by Urigo — APPROVED** [review]

**Comment by Urigo** [comment]
@maticzav the build fails, could you check out why?

**Pull request merged**